### ISSUE TITLE
Propdef grammars should link each component value to its definition

### ISSUE THREAD

**Issue by fantasai** [tabatkins/bikeshed#453]
Compare this version, in which each keyword or function in the propdef links to its property-specific definition
http://dev.w3.org/csswg/css-fonts/#font-variant-prop
to this one, in which only `<valtypes>` get linked to their <valtype> definition
http://dev.w3.org/csswg/css-fonts/Overview.html#font-variant-prop

**Comment by fantasai** [comment]
(This might require additional markup support from the author.)

**Comment by tabatkins** [comment]
There's nothing to do here on the Bikeshed side; it already has all the capabilities necessary.  I just marked it up differently than the original spec did.

**Comment by fantasai** [comment]
No, this is not fixed. E.g. the 'ordinal' keyword in the propdef does not link to its definition, as it does in the non-Bikeshedded version.

**Comment by tabatkins** [comment]
Again:

> There's nothing to do here on the Bikeshed side; it already has all the capabilities necessary. I just marked it up differently than the original spec did.

**Comment by fantasai** [comment]
Can you mark it up in the way the Fonts editor *wants* it to be marked up then?

**Comment by tabatkins** [comment]
We don't do that for any other spec.

**Comment by fantasai** [comment]
I'm not sure what's the problem here. jdaggett's linking scheme (each keyword in the propdef to its definition) is very clearly more useful. This is especially true for the Fonts spec, but I don't see why it wouldn't be more useful anywhere else. It seems to me that at least in most cases, we have enough information to do this linking. No?

**Comment by tabatkins** [comment]
Possibly.  Sometimes the important grammar term is a longer expression than a single keyword, but yeah, single keywords could probably link up automatically.

**Comment by fantasai** [comment]
Right. I think what's needed here for parity is single keywords linking up as well as function names linking up. Is that fair?

**Comment by tabatkins** [comment]
I should be able to do that. Open up a new issue for it?

**Comment by fantasai** [comment]
Um. That's exactly what this issue is about. Why not just reopen it?

**Comment by tabatkins** [comment]
Indeed it is.
